src/components/enter-authenticator-app-code/enter-authenticator-app-code-controller.ts

```diff
@@ -55,9 +59,16 @@ export function enterAuthenticatorAppCodeGet(
     const isAccountRecoveryEnabledForEnvironment = supportAccountRecovery();
 
     if (!isAccountRecoveryEnabledForEnvironment) {
-      return res.render(templateName, {
+      let templateOptions: object = {
         isAccountRecoveryPermitted: false,
-      });
+      };
+      if (showRedactedEmail) {
+        templateOptions = {
+          ...templateOptions,
+          redactedEmail: redactEmail(email),
```

Comment:
Not a huge thing and I'm happy to leave as is. You may be able to save a little verbosity with a ternary here, assuming the template parser understands the undefined variable

e.g. 
```
return res.render(templateName, {
  isAccountRecoveryPermitted: false,
  redactedEmail: showRedactedEmail ? redactEmail(email) : undefined,
});
```